sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/LiveMetrics/Manager.State.cs

```diff
@@ -56,12 +58,30 @@ private void ShutdownState()
             _shutdownEvent.Dispose();
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private void SetPingIntervalFromService(int? milliseconds)
+        {
+            if (milliseconds.HasValue)
+            {
+                if (!_pingPeriodFromService.HasValue
```

Comment:
How do we validate this in the test?

Reply:
we don't have unit tests for the timings. I can write tests to verify that the new period is set

Reply:
Added tests. https://github.com/Azure/azure-sdk-for-net/pull/44383/commits/62b574f219843376cc75272fa29b9163643fc4a5

Needed to make a few smaller changes to facilitate the test.